graders/python/python_autograder/code_feedback.py

```diff
@@ -462,7 +462,7 @@ def call_user(cls, f, *args, **kwargs):
                     "callable."
                 )
 
-            raise GradingComplete() from exc
+            raise GradingComplete from exc
```

Comment:
What's your thinking about when it is vs. isn't appropriate to raise an exception without a message? Both here and in the C grader, we're really just kinda using these for control flow and we'll never try to read the message.

Reply:
If we cannot add additional context to the error message, I think it's fine to raise an exception without a message (it should contain the original exception with `from exc`)